README.md

```diff
@@ -52,13 +52,96 @@ make test-python-tuf
 make test-go-tuf
 ```
 
-It's also possible to locally run the test suite with a client-under-test CLI from another location:
+It's also possible to locally run the test suite with a client-under-test CLI that is locally installed elsewhere:
 
 ```bash
-pip install -e $CONFORMANCE_SUITE_DIR
-pytest "$CONFORMANCE_SUITE_DIR/tuf_conformance" --entrypoint path/to/my/client-under-test/cli
+make dev
+./env/bin/pytest tuf_conformance --entrypoint path/to/my/client-under-test/cli
 ```
 
+### Creating (and debugging) tests
+
+Let's say we want to test that clients will not accept a decreasing targets version. We start with a simple skeleton
+test that doesn't assert anything but sets up a repository and runs client refresh against it:
+
+```python
+def test_targets_version(
+    client: ClientRunner, server: SimulatorServer
+) -> None:
+    # Initialize our repository: modify targets version and make sure the version is included in snapshot
+    init_data, repo = server.new_test(client.test_name)
+    repo.targets.version = 7
+    repo.update_snapshot()  # snapshot v2
+
+    # initialize client-under-test, run refresh
+    client.init_client(init_data)
+    client.refresh(init_data)
+```
+
+we can now run the test:
+```bash
+./env/bin/pytest tuf_conformance \
+    -k test_targets_version                 # run a specific test only
+    --entrypoint "./clients/go-tuf/go-tuf"  # use the included go-tuf client as client-under-test
+    --repository-dump-dir /tmp/test-repos   # dump repository contents
+
+# take a look at the repository targets.json that got debug dumped
+cat /tmp/test-repos/test_targets_version/1/targets.json
+cat /tmp/test-repos/test_targets_version/1/snapshot.json
```

Comment:
Maybe the repository version numbers should mention "refresh" so it's easier to figure out that a version is stored per client refresh... so `/tmp/test-repos/test_targets_version/refresh-1/targets.json` in this example